CODEOWNERS

```diff
@@ -77,7 +77,7 @@ NOTICES     @garyverhaegen-da @dasormeter
 /triggers/service/     @ray-roestenburg-da
 
 # Misc
-/docs/              @stefanobaghino-da
+/docs/              @stefanobaghino-da @digital-asset/daml-docs @hrischuk-da
```

Comment:
@garyverhaegen-da what is the right GH group for the docs team? This one doesn’t seem to have write access. And if we add @hrischuk-da as a reviewer, he should probably also be added to that group or given write access via some product group.

Reply:
`daml-docs` now has write access, as intended.

Reply:
Also created a @digital-asset/daml-product team with @bame-da as maintainer and invited @hrischuk-da to it, and given it write access to this repo.

Reply:
Great thank you! I'll not switch over the code owners to the team since for docs currently only Stefano & Curtis should be notified.